src/org/opendatakit/briefcase/operations/Export.java

```diff
@@ -15,24 +15,6 @@
  */
 package org.opendatakit.briefcase.operations;
 
-import static java.time.format.DateTimeFormatter.ISO_DATE_TIME;
```

Comment:
It seems like this changeset is only about imports ordering. Please, drop this commit.